### PR TITLE
Mepts 739 mq report categoria 10 denominator

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 1]
```diff
@@ -6335,4 +6335,91 @@ public CohortDefinition getMQ9Den(int flag) {
 
     return cd;
   }
+
+  public CohortDefinition getMQ10Den(boolean adults) {
+
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("revisionEndDate", "revisionEndDate", Date.class));
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: Looks like this parameter is NOT applied anywhere in t his cohort definition, it looks redundant for me

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 2]
```diff
@@ -6335,4 +6335,91 @@ public CohortDefinition getMQ9Den(int flag) {
...
+    cd.addSearch(
+        "CHILDREN",
+        EptsReportUtils.map(
+            genericCohortQueries.getAgeOnMOHArtStartDate(0, 14, true),
+            "onOrAfter=${startDate},onOrBefore=${endDate},location=${location}"));
+
+    cd.addSearch(
+        "BABIES",
+        EptsReportUtils.map(
+            genericCohortQueries.getAgeInMonths(0, 20), "effectiveDate=${endDate}"));
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: The specifications says **Age should be calculated on Patient ART Start Date** this age as seen here is based on **reporting end date**

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java [hunk 3]
```diff
@@ -2334,6 +2334,26 @@ public DataSetDefinition constructQualityImprovement2020DataSet() {
             "startDate=${startDate},endDate=${endDate},revisionEndDate=${revisionEndDate},location=${location}"),
         "");
 
+    // MQ indicators category 10 denominator
+    dataSetDefinition.addColumn(
+        "MQ10DEN1A",
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java:2347]: Any special for such naming? why NOT just **MQ10DEN1** we have many more other indicators in the same range that need introduction of that, the other that follow is **MQ10DEN1C**, are we going to reuse this cohort definitions for other upcoming indicators?

**Review by ningosi — COMMENTED** [review]
@cfaife please have a look at the comments for any action required

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java (hunk 3 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java:2347]: if you look at the template there, we have other levels for this category

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 2 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: solved

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 1 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: removed

**Review by ningosi — APPROVED** [review]

**Pull request merged**